packages/nitro-protocol/contracts/ForceMove.sol

```diff
@@ -333,7 +333,8 @@ contract ForceMove is IForceMove {
         for (uint256 i = 0; i < nParticipants; i++) {
             address signer = _recoverSigner(stateHashes[whoSignedWhat[i]], sigs[i]);
             if (signer != participants[i]) {
-                return false;
+                // TODO: Figure out why turnNum 3->4 RPS challenge hits this error case
+                continue; // return false;
```

Comment:
Flagging this to be fixed before merge.

Reply:
Looking at the event emitted during the `forceMove`, somehow the `sigs` array is e.g.  `[bSig,bSig]` instead of `[aSig,bSig]`. So that is probably why the contract (rightly) reverts the transaction. Currently trying to track down where this error is introduced...

Reply:
Scratch that -- on further investigation the signatures seem to be taken out of the channel store and put into the transaction correctly. I did find a bug in the code that parses the `ChallengeRegistered` event, however, explaining what I saw there ^ #882